### DISCUSSION TITLE
Xplat Gen wiki release notes with WI type icon instead of label

### DISCUSSION THREAD

**Discussion by jehandy** [rfennell/AzurePipelines#1027] [Q&A] [answered]
Is there a way to pull the description of the work items with their icons as they appear in ADO so that instead of  this:

  User Story - 24321 Make it easier to cut your toenails
  Task - 25242  Find the nail clippers
  Task - 25243  Find the nail file
  Bug - 22345 Running app causes excessive bellybutton lint

  
We see something like these:

![image](https://user-images.githubusercontent.com/51130285/122795193-10424e00-d28b-11eb-8984-8ce40b514399.png)

**Comment by jehandy** [comment, marked as answer]
And I figured it out. Just need to have the template render the links as below:

#24321 - Make it easier to cut your toenails
#25242 - Find the nail clippers
#25243 - Find the nail file
#22345 - Running app causes excessive bellybutton lint

The wiki in ADO renders them in the format I want. I was over-thinking it.

**Reply by rfennell** [reply]
Glad you are sorted
